docs/api/base_model.md

```diff
@@ -0,0 +1,29 @@
+Pydantic models are simply classes which inherit from `BaseModel` and define fields as annotated attributes.
+
+::: pydantic.BaseModel
+    options:
+        show_root_heading: true
+        merge_init_into_class: false
+        group_by_category: false
+        # explicit members list so we can set order and include `__init__` easily
```

Comment:
You mean that if we add `__init__`, then it's going to be the last?

We need to remember to add methods here when we implement them, right?

Reply:
yes we need to add them.

We can use some very special regex to include `__init__` but not other dunder attributes, but the order is still very confusing. I think since this is a very important bit of the API documentation, it's worth getting it as good as possible even if that means adding members manually.